docs/lib/posixlib.rst

```diff
@@ -10,7 +10,7 @@ Scala Native provides bindings for a core subset of the
 C Header          Scala Native Module
 ================= ==================================
 `aio.h`_          N/A - *indicates binding not available*
-`arpa/inet.h`_    scala.scalanative.posix.arpa.inet_
+`arpa/inet.h`_    scala.scalanative.posix.arpa.inet_ [#inet_ntoa]_
```

Comment:
I think it would be better to point out in general in the interopt section that certain anomalies in the specs of `clib` and `posixlib` exist due to this limitation. We are certainly hopeful that our wizard compiler developers can eventually remove the restrictions.

We can make it more clear at each of the sites like suggested and then when they get changed it will be more obvious and the `scaladoc` can be adjusted when we change them back to spec. Some have multiple places like `complex` and are totally non-standard so we don't want to go overboard so we create more work for later. Plus, we haven't had one question about this yet.

You could add something here above the C and POSIX lib - https://scala-native.readthedocs.io/en/latest/lib/index.html - or maybe here. I don't really see much about the limitation - https://scala-native.readthedocs.io/en/latest/user/interop.html#memory-layout-types

Reply:
Ack.  Will reply when/if I can